lib/Translator.js

```diff
@@ -0,0 +1,248 @@
+'use strict';
+
+const worldStateData = require('warframe-worldstate-data');
+
+/**
+ * Translator to convert in-game names to their localizations
+ */
+class Translator {
+		
+	constructor(lang = 'en_US') {
+		this.data = worldStateData[lang];
+		if(typeof this.data !== "object") {
+			throw new TypeError("data with lang="+lang+" doesn't exists. Please check 'warframe-worldstate-data' dep.");
```

Comment:
To be clearer, maybe not per-function here, but per-function of the object creation so other languages can reside in the same platform-grouped worldstate.